test/xpu_api/iterators/predef.iterators/reverse.iterators/reverse.iter.ops/reverse.iter.op.gtequal/test_rev_iter_op_gteq.pass.cpp

```diff
@@ -68,30 +66,29 @@ kernel_test()
                 ret_access[0] &= test(s, s + 1, true);
                 ret_access[0] &= test(s + 1, s, false);
 
-#if TEST_STD_VER > 14
                 {
                     constexpr const char* p = "123456789";
-                    typedef s::reverse_iterator<const char*> RI;
-                    constexpr RI it1 = s::make_reverse_iterator(p);
-                    constexpr RI it2 = s::make_reverse_iterator(p);
-                    constexpr RI it3 = s::make_reverse_iterator(p + 1);
-                    static_assert((it1 >= it2), "");
-                    static_assert((it1 >= it3), "");
+                    typedef dpl::reverse_iterator<const char*> RI;
+                    constexpr RI it1 = dpl::make_reverse_iterator(p);
+                    constexpr RI it2 = dpl::make_reverse_iterator(p);
+                    constexpr RI it3 = dpl::make_reverse_iterator(p + 1);
+                    static_assert(it1 >= it2);
+                    static_assert(it1 >= it3);
                 }
-#endif
             });
         });
     }
     return ret;
 }
+#endif // TEST_DPCPP_BACKEND_PRESENT
 
 int
-main(int, char**)
+main()
 {
+#ifdef TEST_DPCPP_BACKEND_PRESENT
     auto ret = kernel_test();
-    if (ret)
-        std::cout << "Pass" << std::endl;
-    else
-        std::cout << "Fail" << std::endl;
-    return 0;
+    EXPECT_TRUE(ret, "Wrong result of work in kernel_test()");
```

Comment:
it makes sense to add more details in the message.

Reply:
Fixed